CHANGELOG.md

```diff
@@ -2827,7 +2827,7 @@ restic users. The changes are ordered by importance.
  * Enhancement #3106: Parallelize scan of snapshot content in `copy` and `prune`
 
    The `copy` and `prune` commands used to traverse the directories of snapshots one by one to find
-   used data. This snapshot traversal is now parallized which can speed up this step several
+   used data. This snapshot traversal is now parallelized which can speed up this step several
```

Comment:
The changelog.md is autogenerated from the files in the `changelog/` folder. Those typos have to be fixed there, otherwise they'll just resurface on the next release.

Reply:
Thank you.